src/app/wallet/wallet/balances/balance.component.ts

```diff
@@ -41,19 +43,38 @@ export class BalanceComponent implements OnInit, OnDestroy {
     switch (this.type) {
       case 'total_balance':
         return 'TOTAL BALANCE';
+      case 'actual_balance':
+        return 'SPENDABLE';
       case 'balance':
-        return 'PUBLIC BALANCE';
+        return 'PUBLIC';
       case 'anon_balance':
-        return 'PRIVATE BALANCE';
+        return 'PRIVATE';
       case 'blind_balance':
-        return 'BLIND BALANCE';
+        return 'BLIND';
       case 'staked_balance':
-        return 'STAKE';
+        return 'STAKING';
+      case 'locked_balance':
+        return 'LOCKED';
     }
 
     return this.type;
   }
 
+  listUnSpent(balance: number): void {
+    this._rpcState.observe('listunspent')
+      .pipe(takeWhile(() => !this.destroyed))
+      .subscribe(unspent => {
+          this.availableBal = 0;
```

Comment:
Create a temporary variable of type number here (default assigned a value of 0), then set `this.availableBal` after the for-loop below, using the temp variable to create an Amount type.